src/Tests/CronJobFormTest.php

```diff
@@ -49,6 +50,11 @@ function testManageJob() {
     $this->drupalGet('admin/config/system/cron/jobs');
     $this->assertResponse('200');
 
+    // Check for the default schedule.
+    $this->assertText('Every 15 min', "Job has correct schedule.");
+    // Check that the Cron job has never run.
+    $this->assertText('Never', "Job has not run yet.");
```

Comment:
We're not checking that it wasn't run yet, we're checking how that fact is displayed. No need for the additional messages, just have the assertText() with an improve comment above.